fixposition_driver_ros1/msg/WheelSensor.msg

```diff
@@ -0,0 +1,21 @@
+####################################################################################################
+#
+#    Copyright (c) 2023  ___     ___
+#                       \\  \\  /  /
+#                        \\  \\/  /
+#                         /  /\\  \\
+#                        /__/  \\__\\  Fixposition AG
+#
+####################################################################################################
+#
+# Individual wheelspeed measurement
+#
+#
+####################################################################################################
+
+# Velocity values in [mm/s] as integer 32bit
+# Options:
+# Option 1: One value, will be input in the x axis
+# Option 2: Two values, will be input in the x and y axis
+# Option 2: Three values, will be input in the x, y, and z axis
+int32[] speeds
```

Comment:
These documentation options look a little loose to me. We can IMO be a little more transparent to the FP_B-MEASUREMENTS definition. We can't for example just specify `y` velocity if the sensor is mounted sideways.

Can we switch this to something like:

```suggestion
int32 vx
int32 vy
int32 vz

bool vx_valid
bool vy_valid
bool vz_valid
```
For example: https://docs.ros.org/en/melodic/api/sensor_msgs/html/msg/BatteryState.html